common/pkg/testutils/capitest/patches.go

```diff
@@ -92,6 +92,52 @@ func ValidateGeneratePatches[T mutation.GeneratePatches](
 	DescribeTable("Patches", testFunc, testArgs)
 }
 
+// TODO(shalinpatel): AssertGeneratePatches is a replacement of ValidateGeneratePatches function.
+// remove ValidateGeneratePatches once all the shared test cases are moved to their own patch package.
+func AssertGeneratePatches[T mutation.GeneratePatches](
```

Comment:
This is similar to ValidateGeneratePatches but does not build a table, why the change? Are there any other differences?

Reply:
There is no other changes, it just removes building the table. I plan to remove `ValidateGeneratePatches` or rename `AssertGeneratePatches` to `ValidateGeneratePatches` once all the patches are moved.